README.md

```diff
@@ -29,6 +29,69 @@ npm install cplus
    })
    ```
 
+- You can also pass your own custom transport (option: `transport`).
+
+  ```javascript
+  const myCustomTransportInstance = new CustomTransport(console)
+
+  cplus.install({
+    transport: myCustomTransportInstance
+  })
+  ```
+
+- If you want to use built-in transports and incorporate them into your custom
+  transport, you can access the constructors directly.
+  This is useful if you need to capture console output and work with its
+  data but you want to pass the data to `cplus` console. 
+  You can access the transports with this namespace:
+
+  ```javascript
+  import { transports } from 'cplus'
+
+  // ...
+
+  class CustomTransport {
+    constructor(console) {
+      this._browserTransport = new transports.BrowserTransport(console)
+      this._browserTransport.setLogLevelPrefixes({ 1: ' ! '})
```

Comment:
I'm not sure if it is a good idea to promote this low-level transport construction. It could be mentioned at the very end that the raw constructors are available but I would stick to `createBrowserTransport()` in the examples.

Reply:
will fix it